helm/vertical-pod-autoscaler-app/Chart.yaml

```diff
@@ -11,5 +11,7 @@ sources:
 type: application
 version: [[ .Version ]]
 annotations:
+  repository: https://github.com/FairwindsOps/charts/tree/master/stable/vpa
```

Comment:
As we agreed, adding the upstream repo and the used commit make sense

Reply:
<3 Thanks